lib/view/my_crosswords.dart

```diff
@@ -176,9 +177,8 @@ class ModifyCrosswordNameScreenState extends State<ModifyCrosswordNameScreen> {
                 SizedBox(
                   child: TextField(
                     textAlign: TextAlign.center,
-                    decoration: InputDecoration(
```

Comment:
It should be a text area because the normal user doesn't know that he should click on the text.